api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -142,4 +142,19 @@
     	<dropForeignKeyConstraint baseTableName="test_order" constraintName="test_order_order_id_fk"/>
 		<addForeignKeyConstraint baseColumnNames="order_id" baseTableName="test_order" constraintName="test_order_order_id_fk" deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION" referencedColumnNames="order_id" referencedTableName="orders"/>
 	</changeSet>
+
+	<changeSet id="20200714-Trunk-5862" author="loliveira">
```

Comment:
@dkayiwa @ibacher any preference/convention for the changeset ID?

There seems to be none.

Reply:
This probably needs to be a longer conversation than just comments on a PR. We don't seem to have an established convention and this is at least in line with other IDs, so in that sense, it's fine.

Personally, I would prefer something like: Ticket-Short Description. So, for this case, something like `TRUNK-5862-add_form_namespace_and_path_to_condition`. The idea being that this is unique enough (i.e., every changeset is "namespaced" to a ticket), but descriptive enough that if it appears to an administrator during install, they might understand what failed.

Reply:
Yeah, we should come up with a standard convention, might be worth a Talk post.

But, also, yeah, as we don't currently have a convention so as long as this is in line with existing IDs we can move forward with this one while we figure things out.

Reply:
I also agree with a talk post, but which should not block this pull request from getting merged.